std_msgs/Header header
float64 motor_battery_voltage
float64 electronic_battery_voltage
float64 cable_voltage
bool is_motor_relay_on
bool is_electronic_relay_on
bool is_cable_relay_on